src/screens/Login.tsx

```diff
@@ -1,8 +1,14 @@
 import * as React from 'react';
 import '../stylesheets/loginStyle.css';
 import { Link } from 'react-router-dom'
+import Store from '../store/Store'
+class Login extends React.Component<any, any> {
 
-class Login extends React.Component {
+
+    login(){
+        Store.token = "asdb22222";
```

Comment:
Testing purposes? If so, a comment would be perfect, otherwise if this is a secret key of some sorts, a comment describing its uses :)

Reply:
Made comments on all testing methods